templates/addons/calico-dual-stack/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+    - ../calico-ipv6
+patchesStrategicMerge:
+    - patches/calico-config.yaml
+patches:
+- target: 
+    group: apps
+    version: v1
+    kind: DaemonSet
+    name: calico-node
+    namespace: kube-system
+  path: patches/calico-node.yaml
+- path: patches/control-plane-tolerations.yaml
```

Comment:
is this needed? I believe the ipv6 template already has the tolerations

Reply:
Good point! Let me remove it. I think we can also get rid of `calico-node` because it's the same for ipv6